### DISCUSSION TITLE
Pure Javascript Editor

### DISCUSSION THREAD

**Discussion by vedantroy** [microsoft/monaco-editor#3635] [Q&A]
monaco-editor version:  0.14.3
Browser: Chrome
OS: Linux

Is it possible to make monaco-editor a pure Javascript editor? E.g - disable typescript auto complete and prevent the editor from allowing typescript syntax.

I thought having         `language: 'javascript',` would be enough, but it turns out the editor still provides suggestions like `interface` and allows syntax like `var foo: string` even though that is not valid javascript.

**Comment by nrayburn-tech** [comment]
`var foo: String` errors now.

I am not sure what is meant by interface suggestions still being provided.
